admin/workspace-providers/index.md

```diff
@@ -0,0 +1,85 @@
+---
+title: Workspace Providers
+description: Learn how workspace providers can minimize latency for users.
+state: beta
+---
+
+Workspace providers are logical groups of resources to which developers can
+deploy workspaces. They enable a single Coder deployment to provision and manage
```

Comment:
Everywhere in this document that it says "workspaces" should be "environments", minus "Workspace Providers".